packages/console/src/pages/SignInExperience/components/SignUpAndSignInChangePreview/index.module.scss

```diff
@@ -20,10 +19,12 @@
 
   .section {
     flex: 1;
+    width: calc(50% - _.unit(1.5));
```

Comment:
this looks not stable.. can we just use `50%` or something and let the content grow?

Reply:
We have limits in the confirm modal, so it's hard to grow the width.

Reply:
`50%` is too much because we have `_.unit(3)` between the left and right sections.

Reply:
Updated with new solution.